tasks/lib/PhantomJsHeadlessAnalyzer.js

```diff
@@ -297,7 +298,7 @@ PhantomJsHeadlessAnalyzer.prototype.getDependencies = function (doneFn, task) {
         trace.forEach(function (t) {
             msgStack.push(" -> " + t.file + ": " + t.line + (t["function"] ? " (in function \"" + t["function"] + "\")" : ""));
         });
-        grunt.verbose.error(msgStack.join("\n"));
+        grunt.log.warn(msgStack.join("\n"));
```

Comment:
Using grunt.verbose.error will make the grunt task to exit.
So, I changed this to warn. However, I am researching whether there's a way to just swallow the error and continue...